docs/README.md

```diff
@@ -49,6 +49,40 @@ When running your application, make sure to:
 
 > Some settings can be omitted on .NET (Core). For more information, see the [documentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.3.1-beta.1/docs/config.md#net-clr-profiler).
 
+### Shell scripts
+
+You can install Splunk Distribution of OpenTelemetry .NET
+and instrument your .NET application using the provided Shell scripts.
+Example usage:
```

Comment:
```suggestion
You can install the Splunk Distribution of OpenTelemetry .NET
and instrument your .NET application using the official shell scripts.

For example:
```

Reply:
Can we explain the difference between the install and instrument scripts?

Reply:
How about:

```
The [splunk-otel-dotnet-install.sh](../splunk-otel-dotnet-install.sh) script
installs the OpenTelemetry .NET Automatic Instrumentation.
It uses environment variables as parameters:
```

```
The [instrument.sh](../instrument.sh) script
enables the instrumentation in the current shell session.
It uses environment variables as parameters:
```

I am not sure how to make it clear that "install" only "places the component on local machine".

Reply:
So instrument includes install ? Or they're just separate steps to be run in sequence? If that's the case, what's the reason for splitting them?

Reply:
Will be addressed in a separate PR

Reply:
https://github.com/signalfx/splunk-otel-dotnet/pull/42